index.js

```diff
@@ -210,10 +210,13 @@ var lastResorts = [
   ['BEER', 'Beerware'],
   ['BOOST', 'BSL-1.0'],
   ['BSD', 'BSD-2-Clause'],
+  ['CDDL', 'CDDL-1.1'],
```

Comment:
I did not check for CDDL on npmjs, but I'm using this for other repos, for example maven-cnetral.org (Java repo), where it's much more prevalent. Although npm now encourages us to use SPDX-compliant licenses, SPDX is much bigger than npm :)

About the general issue, I agree the ROI is not clear. It's not a must in my view anyway; maybe a note for future coders of spdx-correct is enough.